gateway-js/src/__tests__/executeQueryPlan.test.ts

```diff
@@ -39,20 +47,26 @@ describe('executeQueryPlan', () => {
 
   let schema: GraphQLSchema;
   let queryPlanner: QueryPlanner;
-
   beforeEach(() => {
-    expect(
-      () =>
-        ({
-          serviceMap,
-          schema,
-          queryPlanner,
-        } = getFederatedTestingSchema()),
-    ).not.toThrow();
+    // expect(
+    //   () =>
+    //     ({
+    //       serviceMap,
+    //       schema,
+    //       queryPlanner,
+    // } = getFederatedTestingSchema()),
+    // ).not.toThrow();
+    let compositionResult = composeAndValidate(fixtures);
+
+    schema = buildComposedSchema(parse(compositionResult.supergraphSdl!));
+    queryPlanner = new QueryPlanner(schema);
+    serviceMap = Object.fromEntries(
+      fixtures.map((f) => [f.name, buildLocalService([f])]),
+    );
```

Comment:
@martijnwalraven I noticed the schema coming back from `getFederatedTestingSchema` was including the `ssn` field so I started by digging in here. I also noticed that taking these same steps in the `printSupergraphSdl` tests weren't having the same issue.

It seems like something is wrong with the fixtures themselves from within that file, but I haven't pinpointed it.

I don't think we should go forward with this but it may be helpful to look at / work backwards from.

Reply:
Resolved in 9cb04ac